backend/tests/v5/e2e/services/chat/modelEvents/modelSettings.test.js

```diff
@@ -219,6 +225,116 @@ const modelDeletedTest = () => {
 	});
 };
 
+const ticketAddedTest = () => {
+	describe('On adding a new ticket', () => {
+		test(`should trigger a ${EVENTS.CONTAINER_NEW_TICKET} event when a new container ticket has been added`, async () => {
+			const socket = await ServiceHelper.socket.loginAndGetSocket(agent, user.user, user.password);
+
+			const data = { teamspace, project: project.id, model: container._id };
+			await ServiceHelper.socket.joinRoom(socket, data);
+
+			const socketPromise = new Promise((resolve, reject) => {
+				socket.on(EVENTS.CONTAINER_NEW_TICKET, resolve);
+				setTimeout(reject, 1000);
+			});
+
+			const newTicket = generateTicket(template);
+			const postRes = await agent.post(`/v5/teamspaces/${teamspace}/projects/${project.id}/containers/${container._id}/tickets/?key=${user.apiKey}`)
+				.send(newTicket)
+				.expect(templates.ok.status);
+
+			const getRes = await agent.get(`/v5/teamspaces/${teamspace}/projects/${project.id}/containers/${container._id}/tickets/${postRes.body._id}?key=${user.apiKey}`)
+				.expect(templates.ok.status);
+
+			await expect(socketPromise).resolves.toEqual({
+				...data,
+				data: {
+					...newTicket,
+					_id: postRes.body._id,
+					properties: {
+						...newTicket.properties,
+						Owner: user.user,
+						'Updated at': new Date(getRes.body.properties['Updated at']).toISOString(),
+						'Created at': new Date(getRes.body.properties['Created at']).toISOString(),
+					},
+					number: 1,
+				},
+			});
+
+			socket.close();
+		});
+
+		test(`should trigger a ${EVENTS.FEDERATION_NEW_TICKET} event when a new federation ticket has been added`, async () => {
+			const socket = await ServiceHelper.socket.loginAndGetSocket(agent, user.user, user.password);
+
+			const data = { teamspace, project: project.id, model: federation._id };
+			await ServiceHelper.socket.joinRoom(socket, data);
+
+			const socketPromise = new Promise((resolve, reject) => {
+				socket.on(EVENTS.FEDERATION_NEW_TICKET, resolve);
+				setTimeout(reject, 1000);
+			});
+
+			const newTicket = generateTicket(template);
+			const postRes = await agent.post(`/v5/teamspaces/${teamspace}/projects/${project.id}/federations/${federation._id}/tickets/?key=${user.apiKey}`)
+				.send(newTicket)
+				.expect(templates.ok.status);
+
+			const getRes = await agent.get(`/v5/teamspaces/${teamspace}/projects/${project.id}/federations/${federation._id}/tickets/${postRes.body._id}?key=${user.apiKey}`)
+				.expect(templates.ok.status);
+
+			await expect(socketPromise).resolves.toEqual({
+				...data,
+				data: {
+					...newTicket,
+					_id: postRes.body._id,
+					properties: {
+						...newTicket.properties,
+						Owner: user.user,
+						'Updated at': new Date(getRes.body.properties['Updated at']).toISOString(),
```

Comment:
these should be numbers (epoch time) , if you need `ISOString` then the ticket data hasn't been serialised.